public/images/types/pipeline.svg

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
I would opt for Financial, the right column, 4th row from the bottom. Alternatively the one on the left of that.